installers/ansible/roles/pgo-preflight/tasks/vars.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Include values.yml
```

Comment:
I think this will always use `ansible/values.yaml` considering it will be defined in the image. Need to add a check so that it picks one `config_path` or the other and doesn't always set it to `{{ playbook_dir }}/values.yaml`